src/Compilers/Core/Portable/PEWriter/PooledBlobBuilder.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Utilities;
+
+namespace Microsoft.Cci
+{
+    internal sealed class PooledBlobBuilder : BlobBuilder
+    {
+        private const int ChunkSize = 1024;
+        private static ObjectPool<PooledBlobBuilder> s_chunkPool = new ObjectPool<PooledBlobBuilder>(() => new PooledBlobBuilder(ChunkSize), 128);
```

Comment:
Let's pull the 128 out into a constant too (it rather gets lost on the end of this long line).